data/korath/korath weapons.txt

```diff
@@ -152,6 +236,82 @@ effect "fire-lance impact"
 	"random angle" 360
 	"velocity scale" 0.
 
+outfit "Korath Blaze-Pike"
+	category "Turrets"
+	cost 1970000
+	thumbnail "outfit/blaze-pike"
+	"mass" 57
+	"outfit space" -54
+	"weapon capacity" -54
+	"turret mounts" -1
+	"required crew" 1
+	weapon
+		sprite "projectile/blaze-pike"
+			"frame rate" 20
+		"hardpoint sprite" "hardpoint/blaze-pike"
+		sound "blaze-pike"
+		"hit effect" "blaze-pike hit"
+		"inaccuracy" 0.45
+		"turret turn" 2.8
+		"velocity" 450
+		"lifetime" 1
+		"reload" 1
+		"firing energy" 5
+		"firing heat" 4.5
+		"shield damage" 7.1
+		"hull damage" 3.1
+		"heat damage" 18
+		"burn damage" 0.45
+	description "The seeming reverence the Korath have for fire appears embodied in this weapon. This pair of Fire-Lances mounted together in a turret do not merely double the gun's effect, but rather magnify it exponentially."
```

Comment:
If this is an exponent it's a very small one, this description feels overblown.
```suggestion
	description "The seeming reverence the Korath have for fire appears embodied in this weapon. This pair of Fire-Lances mounted together in a turret do not merely double the gun's effect but, since the twin contact points constrain the individual dissipation rates, magnify the effect by a proportional amount."
```

Reply:
While I agree you're technically correct, I feel your suggested replacement is too wordy and specific.

Reply:
Perhaps, but it does then explain the relationship that supports the Inferno's balance. The original is also currently on the shorter side of descriptions so far, so it's not like there's not space.

I'm open to alternatives of course, the 'exponentially' thing is the main issue I have.

Reply:
This is the sort of thing where I really wish we could use events to alter outfit descriptions without creating duplicate outfits. If a player hasn't really even met the Korath, just acquired one of these as salvage or something, then this amount of detail should be shortened massively. On the other hand, if they've done the salvage mission set and/or have friendly relations with any of the Korath, then all this detail is entirely appropriate. 

This expanded description sounds entirely like the player is getting their explanations from Taely or Dawn. Which I don't consider a bad thing, it's just unlikely if they haven't *met* Taely or Dawn yet.

On the description at hand: Could this be shortened so that the technical description is removed so it's simply:

> , but rather magnify the effect by a proportional amount>"

(Basically, use the original text for the first part, simply replacing "it exponentially" with "by a proportional amount"

Reply:
On the one hand, I hear you, but on the other, it's a technical explanation that you could take a guess at with what was for me first year university physics. The player is, after all, a ship captain - one of the major things the player character must logically have a good grip on is how heat works, and by the time you're reading this description you must have necessarily got the hang of a lot of different things including but not limited to; custom outfitting, space combat, and boarding. The piercer launcher and many ka het outfits have things that you maybe wouldn't guess at automatically.

Reply:
What about something like:
```suggestion
	description "The seeming reverence the Korath have for fire appears embodied in this weapon. The close proximity of the Fire-Lances mounted on this turret results in a magnification of their effects beyond what two separate Fire-Lances would be capable of."
```

Reply:
That seems like a good compromise between the two.

And yeah, MasterOfGrey does make a good point about understanding of physics and what not. Although I kind of wonder how "commodified" being a ship captain is these days. No mention was made of passing a test or having to do any real maintenance ourselves.

Reply:
@warp-core  I still don't like it -- I find it lacks poetry -- however, it is a lot better than Grey's, and if it keeps everyone from objecting, I don't mind it.